### PR TITLE
Ds 102 change project logo

### LINKED ISSUE #70: Favicon
**Is your feature request related to a problem? Please describe.**
Need to replace Create React App favicon with custom favicon.

**Describe the solution you'd like**
Put favicon in folder and put it in index.html.

### LINKED ISSUE #102: Update pdex project name logo
Updating logo at top of page to something more unique to the project

### PR DESCRIPTION
**Pull request by dannysanchez559**
## Changes
_List Changes Introduced by this PR_
1. update app logo name
2. update header and filterbar png images to svg for better resolution and UI
3. Update favicon icon
4. Remove unnecessary react stock images

Closes #102 
Closes #70

### CODE REVIEW

File: src/components/StyledTabs.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+// // custom style for tab bar appearance
+// const StyledTabs = styled(Tabs)
+//     ({
+//         // change appearance of tab bar- current tab indicator
+//         "& .MuiTabs-indicator": {
+//             backgroundColor: `${pokemonTypeColor}`
+//         }
+//     });
+
+// export default StyledTabs;
```
Comment (Lap343) [inline src/components/StyledTabs.js:14]: Do you guys still need this commented code?

**Review by Lap343 — APPROVED** [review]
Other than the commented code I would say you're good

File: src/components/StyledTabs.js (hunk 1 above)
Comment (dannysanchez559) [reply src/components/StyledTabs.js:14]: Had no idea this was in, garrett just mentioned it! Deleting it and pushing up again as we speak.

**Review by connielion — APPROVED** [review]
Looks good!

**Review by JorgeLVilla — APPROVED** [review]
Nice job Danny!

**Pull request merged**